contracts/zero-ex/README.md

```diff
@@ -67,3 +67,28 @@ yarn lint
 ```bash
 yarn test
 ```
+
+### Run Foundry Tests
```

Comment:
Can we call these integration tests or something more appropriate as just "foundry tests" atm atm includes unit tests which should be runnable without the complex rpc integration setup here.